demo-project/demo-api.raml

```diff
@@ -0,0 +1,88 @@
+#%RAML 1.0
+---
+title: Demo API
+baseUri: http://localhost:8080/
+
+/values:
+    get:
+      description: "Retrieve an array of value entries"
+      responses:
+        200:
+          body:
+            application/json:
+              example: |
+                [
+                    {
+                        "key": "k01",
+                        "value": "some value",
+                        "id": 1
+                    },
+                    {
+                        "key": "k02",
+                        "value": "another value",
+                        "id": 2
+                    }
+                ]
+    post:
+        description: "Create a new value entry"
```

Comment:
Since the uri is `/values`, let's refer to the thing as "value" here as well. So, with implied subject as above, we'd get:

_creates a new value, and returns its ID_